src/components/page/index.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 
+import { Content } from '../content/index';
+
```

Comment:
do you use this component?

Reply:
Old stuff which I added when I experimented with it. I'll remove this,